<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>homePage</class>
 <widget class="QWidget" name="homePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1067</width>
    <height>537</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>番茄钟</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/photos/tomato.png</normaloff>:/photos/tomato.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QPushButton" name="settingPutton">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>40</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;  border: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="adding">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent; border: none;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="positiveTiming">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>430</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;
background-color: transparent;  border: none;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="initialSettingPutton_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;
background-color: transparent; border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>0</y>
     <width>1121</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 290 18pt url(:/font/shangshoujianghushufati.ttf)</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;“每一个番茄钟，都是你接近成功的一步”&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLCDNumber" name="currentTime">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>511</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>settingPutton</zorder>
  <zorder>adding</zorder>
  <zorder>positiveTiming</zorder>
  <zorder>initialSettingPutton_2</zorder>
  <zorder>currentTime</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
